azext_edge/tests/edge/orchestration/resources/connector/opcua/conftest.py

```diff
@@ -172,10 +194,16 @@ def setup_mock_common_responses(
 def assemble_resource_map_mock(
     resource_map_mock: Mock,
     extension: Optional[dict],
-    custom_locations: Optional[List[dict]],
     resources: Optional[List[dict]],
 ):
-    resource_map_mock().custom_locations = custom_locations
-    resource_map_mock().get_resources.return_value = resources
     resource_map_mock().connected_cluster.get_extensions_by_type.return_value = extension
     resource_map_mock().connected_cluster.get_aio_resources.return_value = resources
+
+
+def generate_fortos_object_string(
```

Comment:
nit, we don't have to use codenames, the service is secret sync controller or ssc.